46.md

```diff
@@ -17,7 +17,7 @@ The client always starts by generating a random key which is used to communicate
 The remote signer generates a connection token in the form
 
 ```
-<npub1...>#<optional-secret>?relay=wss://...&relay=wss://...
+bunker://<hex-pubkey>?relay=wss://...&relay=wss://...&secret=<optional-secret>
```

Comment:
Is `bunker` the right name? NIP-46 can be used by all sorts of remote signers, right? not all of them are nsecbunkers.

Reply:
It doesn't have to make sense, it's just what people are using. I like it. When I use it with [nak](https://github.com/fiatjaf/nak) I like to think it refers to `nak bunker`.

Reply:
cool. definitely not a blocker. 🚢